### ISSUE TITLE
The Cortex Folding surgery does not give a mob TRAIT_SPECIAL_TRAUMA_BOOST when performed on them.

### ISSUE THREAD

**Issue by Dezupher** [tgstation/tgstation#50711]
## Reproduction: 
Attempt to perform the Cortex Folding surgery, and the patient recieve no special trait. Additionally, when trying to var edit the TRAIT_SPECIAL_TRAUMA_BOOST in manually with the modify traits window, the trait does not exist on the list of traits, like the other surgery given traits do.

**Comment by ATH1909** [comment]
Also, the cortex folding surgery checks for existing bioware in your ligaments slot, even though the piece of bioware it installs goes in your cortex slot.

**Comment by TemporalOroboros** [comment]
That would explain why I could do the surgery three times in a row and still not remove it from the surgeries list. Also why both cortex surgeries disappeared when I did the ligaments surgery.

**Comment by XDTM** [comment]
the ligaments thing is a bug, but as for the bug reported here the only way i think this could happen is by first doing the cortex imprint surgery and then attempting the cortex folding surgery afterwards, since the bioware datum does check for conflicts before applying its effects

**Comment by ATH1909** [comment]
> the ligaments thing is a bug, but as for the bug reported here the only way i think this could happen is by first doing the cortex imprint surgery and then attempting the cortex folding surgery afterwards, since the bioware datum does check for conflicts before applying its effects

Not in this case, because it checks the ligaments slot instead of the cortex slot.

**Comment by ATH1909** [comment]
This MIGHT be fixed by https://github.com/tgstation/tgstation/pull/50719, depending on why this bug occurred.

**Comment by XDTM** [comment]
> 
> 
> > the ligaments thing is a bug, but as for the bug reported here the only way i think this could happen is by first doing the cortex imprint surgery and then attempting the cortex folding surgery afterwards, since the bioware datum does check for conflicts before applying its effects
> 
> Not in this case, because it checks the ligaments slot instead of the cortex slot.

The surgery does, but the bioware itself has another check as a failsafe, and it has the correct bioware slot. So the surgery can be done, it just won't do anything if there's already another CORTEX bioware present.

**Comment by XDTM** [comment]
Did anyone encounter this bug after #50719 was merged?

**Comment by Spaghetterr** [comment]
bump, this was never fixed. Cortex folding still does nothing.
